assets/js/theme/global.js

```diff
@@ -36,6 +35,5 @@ export default class Global extends PageManager {
         }
         loadingProgressBar();
         svgInjector();
-        objectFitImages();
```

Comment:
Could we be sure that it wont affect any new issues, I believe in some time it was fix for another problem